src/main.zig

```diff
@@ -334,3 +347,18 @@ pub fn main() !void {
         },
     }
 }
+
+var verbose: bool = builtin.mode == .Debug; // In debug mode, force verbose.
+fn logFn(
+    comptime level: std.log.Level,
+    comptime scope: @Type(.EnumLiteral),
+    comptime format: []const u8,
+    args: anytype,
+) void {
+    if (!verbose) {
+        // hide all messages with level greater of equal to debug level.
+        if (@intFromEnum(level) >= @intFromEnum(std.log.Level.debug)) return;
```

Comment:
As `debug` is the last level I think checking an equality should be more clear. But of course like that you can't handle addition lowers levels. So it's also good to me like.

Reply:
Good point

Reply:
But Since we filter here, I do prefer the `>=` to avoid more verbose messages if a lower level is added by Zig later.